LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Can you put Apache license:
https://github.com/NodeFactoryIo/vedran/blob/dev/LICENSE